src/relay/pass/fuse_ops.cc

```diff
@@ -820,6 +823,16 @@ class FuseMutator : private ExprMutator {
         if (new_fields[i].as<VarNode>()) {
           auto copy = Copy(new_fields[i]);
           new_fields.Set(i, copy);
+        } else if (new_fields[i].as<CallNode>()) {
+          size_t j = 0;
+          while (j < i) {
+            if (new_fields[j] == new_fields[i]) break;
+            j++;
+          }
+          if (j != i) {
+            auto copy = Copy(new_fields[i]);
```

Comment:
not sure if this is the best way to fix. Does it work if you remove this [assertion](https://github.com/dmlc/tvm/blob/0806b69e3fb136226fa1dafad00bd2c606cc998d/src/pass/arg_binder.cc#L49)?

Reply:
@vinx13 it works, but should we remove that assertion? For my case, memory planning can pass.

Reply:
it is https://github.com/dmlc/tvm/blob/3441b95e8f1e0435060f2010c10496a4b84973ae/src/codegen/llvm/codegen_llvm.cc#L849
The assertion is not handled properly. We should keep the assertion and fix the codegen.